pages/key-value-pairs/permutations.page.tsx

```diff
@@ -0,0 +1,373 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import KeyValuePairs from '~components/key-value-pairs';
+import ScreenshotArea from '../utils/screenshot-area';
+import SpaceBetween from '~components/space-between';
+import StatusIndicator from '~components/status-indicator';
+import Link from '~components/link';
+import Box from '~components/box';
+import { CopyToClipboard, ProgressBar } from '~components';
+
+export default function KeyValueScenario() {
```

Comment:
This should be using the `PermutationsView` component, see for example https://github.com/cloudscape-design/components/blob/main/pages/column-layout/permutations.page.tsx

Reply:
updated